fastapi/applications.py

```diff
@@ -138,8 +139,8 @@ def add_api_route(
         deprecated: bool = None,
         methods: List[str] = None,
         operation_id: str = None,
-        response_model_include: Set[str] = None,
-        response_model_exclude: Set[str] = set(),
+        response_model_include: Union[SetIntStr, DictIntStrAny] = None,
+        response_model_exclude: Union[SetIntStr, DictIntStrAny] = set(),
```

Comment:
This is the new type for pydantic's `BaseModel.dict`'s `include` and `exclude` arguments